src/crawlee/service_container.py

```diff
@@ -57,7 +62,7 @@ def get_storage_client(*, client_type: StorageClientType | None = None) -> BaseS
         return _services['cloud_storage_client']
 
     if 'local_storage_client' not in _services:
-        _services['local_storage_client'] = MemoryStorageClient()
+        _services['local_storage_client'] = MemoryStorageClient(configuration=configuration)
```

Comment:
why do we even store those two separately? those should be two flavors of a single service, not two different services, at least that's how it was designed in the js version.

Reply:
If you mean why we have a separate `local_storage_client` and `cloud_storage_client`, we do this so that the `force_cloud` switch in the SDK works. In the JS SDK, the `Actor` class holds the cloud client. This feels more readable to me.